buildpack.md

```diff
@@ -209,21 +213,19 @@ If a cache is provided the lifecycle:
 - SHOULD store the Layer Content Metadata associated with each layer so that it can be recovered using the layer Diff ID
 
 Before the next re-build:
-- The lifecycle:
-  - MUST either restore the entire `<layers>/<layer>.toml` file and corresponding `<layers>/<layer>/` directory from any previous build to the same paths or
-  - MUST restore neither the `<layers>/<layer>.toml` file nor corresponding `<layers>/<layer>/` directory.
+- The lifecycle MUST do both or neither of the following:
+  - Restore Layer Content Metadata to `<layers>/<layer>.toml`, excluding the `launch`, `build`, and `cache` booleans.
+  - Restore layer contents to the `<layers>/<layer>/` directory.
+
+#### Ignored Layers
 
-#### Other Layers
+Layers marked `launch = false`, `build = false`, and `cache = false` behave like temporary directories, available only to the authoring buildpack, existing for the duration of a single build.
 
-The lifecycle:
-- If `build = false`
-  - MUST NOT make the layer available to subesequent buildpacks.
-- If `launch = false`
-  - MUST NOT include the layer or the Layer Content Metadata in the OCI image.
-- If `cache = false`
-  - MUST NOT store the layer or the Layer Content Metadata in the cache.
+At the end of each individual buildpack's build phase:
+- The lifecycle:
+  - MUST rename `<layers>/<layer>/` to `<layers>/<layer>.ignore/` for all layers where `launch = false`, `build = false`, and `cache = false`.
```

Comment:
Do we need to state how the lifecycle treats layers with a `.ignore` suffix? An interpretation of this could suggest that `<layer>.ignore.toml` controls the behavior of the ignored layer, when really we're just trying to get the layer out of the way / change its path.

Reply:
good point, I'll update this

Reply:
Hmm, given we only rename to `<layer>.ignore` when all of `launch`, `build`, and `cache` are `false`, technically the lifecycle doesn't need to treat a layer with an `.ignore` suffix any differently than a regular layer. The lifecycle already does nothing with this type of layer.

However, I can add a sentence to clear up the intent behind the rename here.